### PR TITLE
Bjorncs/vespa mtls

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: en/setting-vespa-variables.html [hunk 1]
```diff
@@ -44,6 +44,10 @@
     Also note that this needs to be set when starting the <em>configserver</em>,
     since the REST api implementation gets its port number from there.
   </td></tr>
+<tr><th>VESPA_TLS_CONFIG_FILE</th>
+  <td>
+    Path to <a href="mtls.html">TLS configuration file</a>.
```
Comment (vekterli) [inline en/setting-vespa-variables.html:49]: Consider s/Path/Absolute path/ for added clarification

File: en/mtls.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+title: "Securing Vespa with mutually authenticated TLS (mTLS)"
+---
+
+## Introduction
+{% include note.html content="See accompanying [reference](reference/mtls.html) for details on for instance configuration syntax." %}
+{% include note.html content="This document is only relevant if you're **self-hosting Vespa**.
+If you're using Vespa Cloud all services are automatically set up securely by default with full mTLS,
+with all key/certificate management handled for you.
+" %}
+{% include note.html content="Though the terms *TLS* and *mTLS* may be used interchangeably in this document, *TLS* implies *mTLS* for all cluster-internal traffic." %}
```
Comment (vekterli) [inline en/mtls.md:12]: As discussed offline, these notes should probably be converted into paragraphs inline with the text itself

File: en/mtls.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+
+Enabling TLS in Vespa means that all internal endpoints are mTLS protected,  even HTTP servers for status pages and metrics.
+Be especially aware of this if you have custom solutions in place for collecting
+and aggregating low level metrics or status pages from the Vespa backends.
+
+## Prerequisites
+
+{% include important.html content="This document assumes you have some experience with generating and
+using certificates and private keys.
+" %}
```
Comment (vekterli) [inline en/mtls.md:47]: Consider inlining this instead of having it be a warning

File: en/reference/mtls.md [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+    </tr>
+    <tr>
+        <td>accepted-ciphers</td>
+        <td>No</td>
+        <td>JSON array of accepted TLS cipher suites.</td>
+    </tr>
+    <tr>
+        <td>accepted-protocols</td>
+        <td>No</td>
+        <td>JSON array of accepted TLS protocol versions.</td>
```
Comment (vekterli) [inline en/reference/mtls.md:63]: Add link to section for defaults for both `accepted-ciphers` and `accepted-protocols`

**Review by vekterli — APPROVED** [review]
:+1:

**Review by tokle — APPROVED** [review]

**Review by tokle — APPROVED** [review]

**Pull request merged**